policy/modules/services/xserver.te

```diff
@@ -739,6 +739,7 @@ systemd_status_power_services(xdm_t)
 systemd_hwdb_mmap_config(xdm_t)
 systemd_hwdb_read_config(xdm_t)
 systemd_coredump_domtrans(xdm_t)
+systemd_login_watch_session_dirs(xdm_t)
```

Comment:
This has already been addressed with commit 0896c2b06d39f47ff20cff7e2cf72d6d8598eae0.